### ISSUE TITLE
Atom is not updating itself

### ISSUE THREAD

**Issue by marcosousapoza** [atom/atom#17184]
I am new to Atom and have now ben confronted with my first issues.
I do not know much about Server connections and Sockets but the error should, according to the error message, lie exactly there:
![unbenannt](https://user-images.githubusercontent.com/38554157/39043785-98559196-448e-11e8-9856-e312860b7a2c.JPG)
Error = [1524210989.984][SEVERE]: bind() returned an error: Normally each socket 
            address (protocol, network address or connection) may only be used once. 
            (0x2740)
The next things that stopped working are the packages:
![unbenannt](https://user-images.githubusercontent.com/38554157/39043892-e22da16e-448e-11e8-93d5-e9d3e8debfd1.JPG)
I assumed this is somehow bound together...
Does someone know how to fix this?
The OS I am using is Windows 10

**Comment by rsese** [comment]
Thanks for the report - can you share more information about the issue? In particular, the issue template information is really helpful:

* Can you reproduce in safe mode (atom --safe)?  
* Have you tried any of the other [debugging suggestions](https://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* What version of Atom are you running (`atom -v`)?
* Are you only having trouble updating auto-complete-python or with any package?

If you can share the above details, we can re-open and go from there.

---

We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

Thanks for understanding and meeting us half way 😀

**Comment by marcosousapoza** [comment]
So I dug for some more information and here is what I have found:

- The Atom version I am using is Atom 1.26.0
- When I use safe mode the same errors occur
- The error is only caused by the auto-complete-package, but with updating, I have generally a problem.
- I cannot search for updates because then again it shows the error: Error = [1524210989.984][SEVERE]: bind() returned an error: Normally each socket
address (protocol, network address or connection) may only be used once.
(0x2740)
- I tried resetting atom and reinstall the packages which gave me another error:

![unbenannt](https://user-images.githubusercontent.com/38554157/39081589-65dfc892-4544-11e8-85d3-91839d97f5b4.JPG)

I did not have this when I first install atom.
